pages/common/yt-dlp.md

```diff
@@ -22,16 +22,16 @@
 
 - Specify audio format and audio quality of extracted audio (between 0 (best) and 10 (worst), default = 5):
 
-`yt-dlp --extract-audio --audio-format {{mp3}} --audio-quality {{0}} "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
+`yt-dlp --extract-audio --audio-format {{mp3}} --audio-quality {{0}} "{{url}}"`
 
 - Download all playlists of YouTube channel/user keeping each playlist in separate directory:
 
-`yt-dlp -o "{{%(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{https://www.youtube.com/user/TheLinuxFoundation/playlists}}"`
+`yt-dlp -o "{{%(uploader)s/%(playlist)s/%(playlist_index)s - %(title)s.%(ext)s}}" "{{url}}"`
 
 - Download Udemy course keeping each chapter in separate directory under MyVideos directory in your home:
 
-`yt-dlp -u {{user}} -p {{password}} -P "{{~/MyVideos}}" -o "{{%(playlist)s/%(chapter_number)s - %(chapter)s/%(title)s.%(ext)s}}" "{{https://www.udemy.com/java-tutorial}}"`
+`yt-dlp -u {{user}} -p {{password}} -P "{{~/MyVideos}}" -o "{{%(playlist)s/%(chapter_number)s - %(chapter)s/%(title)s.%(ext)s}}" "{{url}}"`
 
 - Download entire series season keeping each series and each season in separate directory under C:/MyVideos:
 
-`yt-dlp -P "{{C:/MyVideos}}" -o "{{%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s}}" "{{https://videomore.ru/kino_v_detalayah/5_sezon/367617}}"`
+`yt-dlp -P "{{C:/MyVideos}}" -o "{{%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s}}" "{{url}}"`
```

Comment:
```suggestion
`yt-dlp -P "{{path/to/directory}}" -o "{{%(series)s/%(season_number)s - %(season)s/%(episode_number)s - %(episode)s.%(ext)s}}" "{{url}}"`
```

Please see https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths

Reply:
FYI, this was marked as resolved without any commits being added.
Do not close comments from maintainers without discussing or doing the suggested change.